src/Database/Traits/Validation.php

```diff
@@ -373,6 +373,12 @@ protected function processValidationRules($rules)
              * Analyse each rule individually
              */
             foreach ($ruleParts as $key => $rulePart) {
+                /*
+                 * Rule class instances should not be checked here
+                 */
+                if(is_object($rulePart)) {
```

Comment:
@filipac This should check that the `$rulePart` is a `Rule` object, and not just any arbitrary object.

Also, please add a space between the `if` and opening bracket. :)

Reply:
Afaik any class can be used there, not only those implementing Rule. Am i wrong? I am on the mobile now

Reply:
@filipac All objects were being skipped before your change, so I don't believe so. I would rather us ensure only `Rule` objects are used instead of string values just to keep consistency with Laravel.